backend/dal/Migrations/20200401221110_initial.Designer.cs

```diff
@@ -10,8 +10,8 @@
 namespace Pims.Dal.Migrations
 {
     [DbContext(typeof(PimsContext))]
-    [Migration("20200329131456_Initial")]
-    partial class Initial
+    [Migration("20200401221110_initial")]
+    partial class initial
     {
```

Comment:
We already spent some time on a previous PR to make sure the  `PostDeploy/Initial` folder was PascalCase (ie - Initial vs "initial") as this creates issues in Linux where our docker containers are built. 

Please change the name back to `*_Initial` as well as the `[Migration]` attribute. I believe that's what we use to get the path to the seed *.sql scripts...

But please @Fosol  comment on this as you created this.